ts/features/mvl/screens/MvlRouterScreen.tsx

```diff
@@ -2,17 +2,18 @@ import * as pot from "italia-ts-commons/lib/pot";
 import * as React from "react";
 import { NavigationInjectedProps } from "react-navigation";
 import { useIODispatch, useIOSelector } from "../../../store/hooks";
+import { UIMessageId } from "../../../store/reducers/entities/messages/types";
 import { useOnFirstRender } from "../../../utils/hooks/useOnFirstRender";
 import { mvlDetailsLoad } from "../store/actions";
 import { mvlFromIdSelector } from "../store/reducers/byId";
-import { MvlData, MvlId } from "../types/mvlData";
+import { Mvl } from "../types/mvlData";
 import { MvlGenericErrorScreen } from "./ko/MvlGenericErrorScreen";
 import { MvlDetailsScreen } from "./MvlDetailsScreen";
 import { MvlLoadingScreen } from "./MvlLoadingScreen";
 
 type NavigationParams = Readonly<{
   // TODO: assumption, we have an unique id that we should use to retrieve the MVL, maybe this could be the messageId? let's see!
```

Comment:
is this TODO always valid?

Reply:
Thank you, no, we have the answer at this question now :D removed in d38f816c293548bc858ccdab81b3183e2c0270f2